virtualpdu/main.py

```diff
@@ -26,6 +27,8 @@
 
 MISSING_CONFIG_MESSAGE = 'Missing configuration file as first parameter.\n'
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
you should consider doing this before any other import as they might instantiate their logger in the module and if it's instantiated before it might not pick up this change to the root logger